miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java

```diff
@@ -358,22 +522,47 @@ private void testTabbedColumnsSort(String listTarget) {
     Set<String> tabHeadings = page.getTabHeadings();
     tabHeadings.forEach(tabHeading -> {
       page.clickTab(tabHeading);
-      assertTrue("clicked tab without errors", LimsUtils.isStringEmptyOrNull(page.getErrors().getText()));
+      assertTrue("clicked tab without errors", isStringEmptyOrNull(page.getErrors().getText()));
     });
   }
 
+  private void testSearch(String listTarget, String searchTarget) {
+    // confirm that searching by a term returns no errors
+    ListPage page = getList(listTarget);
+    page.getTable().searchFor(searchTarget);
+    assertTrue("error searching " + listTarget + " page for '" + searchTarget + "': ", isStringEmptyOrNull(page.getErrors().getText()));
```

Comment:
This is testing that the page doesn't politely inform you of a failure, but not that the search actually worked as expected. Should be testing that an expected record was found, and an unexpected record wasn't. I don't think there's a one-size-fits-all method for this

Reply:
As per our conversation, I've made a ticket to write unit tests for the Pagination Filters, as unit tests will run faster and can be less brittle.

Reply:
:+1: